src/room_member.cpp

```diff
@@ -175,8 +175,7 @@ void RoomMember::ReceiveLoop() {
 };
 
 void RoomMember::Join(const std::string& nickname, const std::string& server, uint16_t server_port, uint16_t client_port) {
-    // TODO(Subv): Use client_port.
-    RakNet::SocketDescriptor socket;
+    RakNet::SocketDescriptor socket(client_port, 0);
```

Comment:
`nullptr` *IS* the same as 0 in this case. `nullptr` is (afaik) meant as a drop in replacement for `NULL` which is defined as `#define NULL (void*)0`.

However, I feel like we can go through and fix these later because we have so many other occurrences in the codebase. It's one of these: Don't turn a "haystack made out of needles" into "a haystack made of hay with just one needle".